### PR TITLE
Fix init sequence where CNIs are not complete with IP address allocation

### PR DESCRIPTION
**Pull request by dstiliadis**
Fixes the initialization of the local interfaces and makes sure we recover from this even if the CNIs are not ready when we are ready.

### CODE REVIEW

**Comment by dstiliadis** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "trireme-lib",
    "pr-id": "889",
    "commit-sha": "936de0ccc6a56d1bfa66aae9eddad69cabb6498b"
  },
  {
    "project": "",
    "component": "enforcerd",
    "pr-id": "1426",
    "commit-sha": "cd6f9d72be3bea8be8b7e94b8bcbd6c3d305f8d8"
  }
]
```

**Comment by dstiliadis** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "enforcerd",
    "pr-id": "1426",
    "commit-sha": "cd6f9d72be3bea8be8b7e94b8bcbd6c3d305f8d8"
  },
  {
    "project": "",
    "component": "trireme-lib",
    "pr-id": "889",
    "commit-sha": "c81d8458d12e9fe3d20c16f13897a326e7d44d61"
  }
]
```

File: controller/internal/enforcer/applicationproxy/protomux/protomux.go [hunk 1]
```diff
@@ -160,6 +161,10 @@ func (m *MultiplexedListener) Serve(ctx context.Context) error {
 			return nil
 		case <-m.shutdown:
 			return nil
+		case <-time.After(5 * time.Second):
```
Comment (mheese) [inline controller/internal/enforcer/applicationproxy/protomux/protomux.go]: should be a ticker instead of time.After

**Review by mheese — APPROVED** [review]
Ideally it should be a ticker instead of time.After, but that's nitpicking :)

**Comment by dstiliadis** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "trireme-lib",
    "pr-id": "889",
    "commit-sha": "c81d8458d12e9fe3d20c16f13897a326e7d44d61"
  },
  {
    "project": "",
    "component": "enforcerd",
    "pr-id": "1426",
    "commit-sha": "78fb33586d12033cd268d0ad7a4edcdc28ac9110"
  }
]
```

**Comment by dstiliadis** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "enforcerd",
    "pr-id": "1426",
    "commit-sha": "9e963c9c6d03c6451d0de53a1a5df32eec5a426e"
  },
  {
    "project": "",
    "component": "trireme-lib",
    "pr-id": "889",
    "commit-sha": "c81d8458d12e9fe3d20c16f13897a326e7d44d61"
  }
]
```

**Comment by dstiliadis** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "trireme-lib",
    "pr-id": "889",
    "commit-sha": "c81d8458d12e9fe3d20c16f13897a326e7d44d61"
  },
  {
    "project": "",
    "component": "enforcerd",
    "pr-id": "1426",
    "commit-sha": "9e963c9c6d03c6451d0de53a1a5df32eec5a426e"
  }
]
```

**Comment by dstiliadis** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "trireme-lib",
    "pr-id": "889",
    "commit-sha": "c81d8458d12e9fe3d20c16f13897a326e7d44d61"
  },
  {
    "project": "",
    "component": "enforcerd",
    "pr-id": "1426",
    "commit-sha": "9e963c9c6d03c6451d0de53a1a5df32eec5a426e"
  }
]
```

**Comment by dstiliadis** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "trireme-lib",
    "pr-id": "889",
    "commit-sha": "8ac9e892e9588aa445a3386202799eba6589fed1"
  },
  {
    "project": "",
    "component": "enforcerd",
    "pr-id": "1426",
    "commit-sha": "4c7e49e3ba13e114af1ddb94b53e7c5345a7faa8"
  }
]
```

**Comment by dstiliadis** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "trireme-lib",
    "pr-id": "889",
    "commit-sha": "8ac9e892e9588aa445a3386202799eba6589fed1"
  },
  {
    "project": "",
    "component": "enforcerd",
    "pr-id": "1426",
    "commit-sha": "d72e6e79cf5136ccc111256398f5fe6cef30e65f"
  }
]
```

**Comment by dstiliadis** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "trireme-lib",
    "pr-id": "889",
    "commit-sha": "8ac9e892e9588aa445a3386202799eba6589fed1"
  },
  {
    "project": "",
    "component": "enforcerd",
    "pr-id": "1426",
    "commit-sha": "d105e0fc0725fd7f878ac4a6c353798a6a5d3c97"
  }
]
```

**Comment by dstiliadis** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "trireme-lib",
    "pr-id": "889",
    "commit-sha": "8ac9e892e9588aa445a3386202799eba6589fed1"
  },
  {
    "project": "",
    "component": "enforcerd",
    "pr-id": "1426",
    "commit-sha": "f1321fd165440b6e56a1fd9709b31bf1f150d13a"
  }
]
```

**Comment by dstiliadis** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "trireme-lib",
    "pr-id": "889",
    "commit-sha": "8ac9e892e9588aa445a3386202799eba6589fed1"
  },
  {
    "project": "",
    "component": "enforcerd",
    "pr-id": "1426",
    "commit-sha": "05ffdcd11bc73adccde954b641591ad327ee2b73"
  }
]
```

**Comment by dstiliadis** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "trireme-lib",
    "pr-id": "889",
    "commit-sha": "8ac9e892e9588aa445a3386202799eba6589fed1"
  },
  {
    "project": "",
    "component": "enforcerd",
    "pr-id": "1426",
    "commit-sha": "6dd85e0d06c3660d7672d801ae2ae838491a9b3a"
  }
]
```

**Pull request merged**